src/renderer/components/+events/events.tsx

```diff
@@ -102,7 +102,7 @@ export class Events extends React.Component<Props> {
             },
             event.getNs(),
             kind,
-            <Link to={detailsUrl} title={name} onClick={stopPropagation}>{name}</Link>,
+            detailsUrl ? <Link to={detailsUrl} title={name} onClick={stopPropagation}>{name}</Link> : name,
```

Comment:
Maybe there are other places too? Why it's fixed like this?

Reply:
Maybe the link should be always as "details-link" and handle this exception inside `kube-object-details.tsx`?